addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-config-vars-service.js

```diff
@@ -249,6 +249,21 @@ class EnvironmentConfigVarsService extends Service {
       });
     }
 
+    // TODO: If the ami sharing gets moved (because it doesn't contribute to an env var)
```

Comment:
This comment section was accidentally removed in this [PR](https://github.com/awslabs/service-workbench-on-aws/pull/575/files). 
Adding it back